LayoutTests/platform/glib/TestExpectations

```diff
@@ -4841,6 +4836,10 @@ imported/w3c/web-platform-tests/selection/caret/move-around-generated-content.ht
 # Needs rebaseline.
 fast/text/glyph-display-lists [ Skip ]
 
+# Now that partial fonts are working, two characters previously rendered as a square are now showing correctly, which makes the test fail. 
+# Probably the test should be updated.
+fast/text/text-combine-placement.html [ Failure ]
```

Comment:
Can you add this to the windows TestExpectations as well? Fails there now for what looks like similar reasons.

Reply:
Thanks, I appreciate it!